cmd/main.go

```diff
@@ -142,7 +165,18 @@ func main() {
 		zap.Duration("stream_success_period_auth", envConfig.StreamSuccessPeriod.Auth),
 	)
 
-	// Start the gops agent
+	// List pods from default namespace
+	pods, err := clientset.CoreV1().Pods("illumio-cloud").List(context.TODO(), metav1.ListOptions{})
```

Comment:
The namespace is not hardcoded anymore.
Use `envConfig.PodNamespace` like the rest of the code.